libs/dh/admin/feature-user-management/src/lib/feature-create-userrole/tabs/create-role-permissions-tab/dh-create-userrole-permissions-tab.component.scss

```diff
@@ -0,0 +1,25 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+:host {
+  display: block;
+  // TODO: Add spacing variable for 24px
+  margin: 24px var(--watt-space-s);
```

Comment:
Suggestion: Talk with @dzhavat about getting that 24px added to the design system :-)

Reply:
Will do